jsonschema2pojo-integration-tests/pom.xml

```diff
@@ -88,6 +88,18 @@
         </plugins>
     </build>
 
+    <dependencyManagement>
+        <dependencies>
+          <dependency>
+              <groupId>org.jetbrains.kotlin</groupId>
+              <artifactId>kotlin-bom</artifactId>
```

Comment:
Why are there now kotlin artifacts being introduced in the integration tests?

Reply:
If not specified, enforcer RequireUpperBoundDeps failed due to the Moshi dependency

Reply:
I see you added some notes about this.  Thanks.